go/rpc/network_client.go

```diff
@@ -3,12 +3,14 @@ package rpc
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
 const (
-	ws = "ws://"
+	ws   = "ws://"
```

Comment:
How do you feel about https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_ ?

Reply:
Fine by me, but would be hard to enforce (maybe we can get the linter to complain?). In any case, don't think it makes sense to fix it in this PR for a single constant.